modules/node-tuning-operator.adoc

```diff
@@ -19,12 +23,24 @@ endif::operators[]
 ifdef::perf[]
 = About the Node Tuning Operator
 endif::perf[]
+ifdef::cluster-caps[= Node Tuning capability]
+
 ifdef::operators[]
 [discrete]
 == Purpose
 endif::operators[]
+
+ifdef::cluster-caps[]
+The Node Tuning Operator provides the features for the `NodeTuning` capability.
+endif::cluster-caps[]
+
 The Node Tuning Operator helps you manage node-level tuning by orchestrating the TuneD daemon and achieves low latency performance by using the Performance Profile controller. The majority of high-performance applications require some level of kernel tuning. The Node Tuning Operator provides a unified management interface to users of node-level sysctls and more flexibility to add custom tuning specified by user needs.
 
+ifdef::cluster-caps[]
+If you disable the NodeTuning capability, some tuning settings will not be applied to the control-plane Nodes which may limit the scalability and performance of large clusters with over 900 Nodes or 900 Routes.
```

Comment:
The overall content looks good to me, but there is a little confusion in below content. 

If you disable the NodeTuning capability, some tuning settings will not be applied to the control-plane Nodes which may limit the scalability and performance of large clusters with over 900 Nodes or 900 Routes.

This should be <be applied to the control-plane.> or <control-plane Nodes>? missing a period '.'

<large clusters with over 900 Nodes or 900 Routes. > should be <large clusters with over 900 nodes or 900 routes.>, does we should keep first letter capital for **Node** and **Route**

Reply:
The original sentence totally makes sense to me and I believe the capitalization is correct.  However, I can imagine we can perhaps split it into two smaller sentences and end the first one with "applied to the control-plane Nodes." to make the content easier to interpret.

Reply:
I will confirm as part of the peer review process, but I think **nodes** and **routes** can be lower case. https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-formatting